### DISCUSSION TITLE
Samples and Results not similar

### DISCUSSION THREAD

**Discussion by Gor80hd** [d8ahazard/sd_dreambooth_extension#767] [Q&A]
I started training in Dreamboot LoRA everything went fine, he completed the training at 15,000 and after downloading the model everything worked correctly. After that, I continued my training by choosing the same model for 15,000 at the LoRA point. The training went on, in the samples folder, I saw the result. After completing the second stage of training, I also selected the latest model, converted, and entered the desired promt into txt2img. But a completely different picture turned out, which is issued in the original 1.5, as if the neural network.
Sample folder during training:
![Снимок экрана_20230111_062336](https://user-images.githubusercontent.com/92448318/212137188-fa33aa32-249b-4d13-82fb-a3dd265896c9.png)
had forgotten everything. I tried using the penultimate option, but it turned out the same. I don't know what to do. Please help me!

txt2img tab after training:
![Снимок экрана_20230111_062324](https://user-images.githubusercontent.com/92448318/212137267-e4eea2e0-605f-4598-af79-60952b4d0cef.png)

**Comment by ExponentialML** [comment]
Hey @Gor80hd . Could you try selecting the LoRA file, *save your settings*, then compiling the checkpoint?
